dashboard/app/helpers/plc/learning_modules_helper.rb

```diff
@@ -3,10 +3,13 @@ def options_for_plc_task_learning_modules
     Plc::LearningModule.all.pluck(:name, :id).sort
   end
 
-  def options_for_evaluation_answer_modules(course_unit)
-    course_unit.plc_learning_modules.order(:required, :name).pluck(:id, :required, :name).map do |id, required, name|
-      # Boo Ruby, you can't add nil to a string without choking?
-      ["#{name}#{' - Required' if required}", id]
+  def options_with_learning_module_names_and_required_tag(course_unit)
+    course_unit.plc_learning_modules.order(:required, :name).map do |learning_module|
+      [learning_module.get_name_with_required_tag, learning_module.id]
     end
   end
+
+  def learning_modules_select_size options
+    [options.size, 25].min
+  end
```

Comment:
Nit: From my earlier feedback, prefer a `max_options_select_tag` helper like [`boolean_check_box`](https://github.com/code-dot-org/code-dot-org/blob/d29dcf90cf7654371dcede0f39c1d622d9f5bfa6/dashboard/app/helpers/levels_helper.rb#L491-L493).  Or better yet, maybe a LearningModule-specific select tag helper.

I think we can further de-dupe these similar tags:
```ruby
= select_tag 'learning_module_ids', options_for_select(options, selected: @enrolled_modules.pluck(:id), disabled: course_unit.get_required_learning_module_ids), {multiple: true, size: learning_modules_select_size(options), style: 'width: 100%'}
```
```ruby
= f.select :plc_learning_module_ids, options_for_select(options, @task.plc_learning_modules.pluck(:id)), {}, {multiple: true, size: learning_modules_select_size(options), style: 'width: 100%'}
```
```ruby
= f.select :plc_learning_module_ids, options_for_select(options, @task.plc_learning_modules.map(&:id)), {}, {multiple: true, size: [options.size, 25].min, style: 'width: 100%'}
```
I'm not sure how much of a win it is to extract `[options.size, 25].min` to a helper (or why passing `options` is preferable to passing `size`).